DNN Platform/DotNetNuke.Abstractions/Logging/ILogTypeInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace DotNetNuke.Abstractions.Logging
+{
+    /// <summary>
+    /// Log type info.
+    /// </summary>
+    public interface ILogTypeInfo
+    {
+        /// <summary>
+        /// Gets or sets log type css class.
+        /// </summary>
+        string LogTypeCSSClass { get; set; }
```

Comment:
```suggestion
        string LogTypeCssClass { get; set; }
```

Reply:
Fixed in latest code push, resolving thread